src/System.Device.Gpio/Interop/Unix/Libc/Interop.ioctl.cs

```diff
@@ -10,6 +10,10 @@ internal partial class Interop
     [DllImport(LibcLibrary, SetLastError = true)]
     internal static extern int ioctl(int fd, uint request, IntPtr argp);
 
+
+    [DllImport(LibcLibrary, SetLastError = true)]
+    internal static extern int ioctl(int fd, int request, IntPtr argp);
```

Comment:
can you remove the previous overload since per `ioctl(2)` it's unsigned long anyway so I presume it will be casted

Reply:
Request contains negative numbers

Reply:
you can use `(uint)-1` - it should have the same effect